app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -42,25 +42,69 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  swagger_schema :GetHighlights do
+    key :required, [:source_type]
+    property :page do
+      key :type, :integer
+      key :description, 'The page number'
+    end
+    property :per_page do
+      key :type, :integer
+      key :description, 'The number of highlights to return per page'
+    end
+    property :order do
+      key :type, :string
+      key :enum, %w[asc desc]
+      key :description, 'The returned sorted order'
+    end
+  end
+
   swagger_schema :Highlight do
     key :required, [:id]
   end
 
+  swagger_schema :Highlights do
+    property :highlights do
+      key :type, :array
+      key :description, 'One or more highlights'
+      items do
+        key :'$ref', :Highlight
+      end
+    end
+  end
+
   swagger_schema :NewHighlight do
     key :required, [:source_type, :source_id, :anchor, :highlighted_content, :color, :location_strategies]
   end
 
+  add_properties(:GetHighlights, :NewHighlight, :Highlight) do
```

Comment:
Not sure if `GetHighlights` should share these properties, because the descriptions need to be different.  E.g. for `:source_parent_ids` in `Highlight` and `NewHighlight`, the field is for all of the parent IDs of the highlight whereas in `GetHighlight` the description should be something like "One or more unrelated parent IDs; query results will have at least one parent ID that matches those provided"

Ditto the others a little, e.g. "The source_type of the highlight" vs "The source_type to limit results to"